client/src/NavBar.jsx

```diff
@@ -0,0 +1,12 @@
+export default function NavBar() {
```

Comment:
Good start! For this project, this is okay! However, if you want to challenge yourself, how would we make this reusable?